### PR TITLE
Performance audit

### PR DESCRIPTION
**Pull request by chiztechnology**
### Hello, dear review. 🎊 

In this PR we focused on learning and applying audit performance on our `vet's database`.
> After adding a bunch of data (over 3.4 million visits and animals); we figured out that the server response was taking a long to send the response.

Please find in these screenshots below a clear understanding of the situation; 👇 

In this first image; we just added random records to our database ;

image.

The screenshot below shows the query execution time 👇 
image

image

Then after working on optimizing the database performance; here is the amazing result that we got 😊 🎊 

image

image

image

Kindly check the commit history for all the logs and necessary information that are not described in this Comment.

Best regards.

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -51,3 +51,24 @@ CREATE TABLE visits(
...
+CREATE INDEX email_asc ON owners(email ASC);
+
+-- create index on animals id
+ CREATE INDEX animals_id_asc ON visits(animals_id ASC);
+
+ CREATE INDEX vets_id_asc ON visits(vets_id ASC);  
+
+ EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animals_id = 4;
+ EXPLAIN ANALYZE SELECT * FROM owners where email = 'owner_18327@mail.com';
+ EXPLAIN ANALYZE SELECT * FROM visits where vet_id = 2;
```
Comment (alexander16108) [inline schema.sql:74]: as required there should be screenshots for three queries showing before and after query execution time. but currently, there are only one showing the before and after queries.
show the screenshots for the following queries to move ahead

`
EXPLAIN ANALYZE SELECT * FROM visits where vet_id = 2;
`
`
EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animals_id = 4;
`

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @chiztechnology , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- Gitflow was correctly used. ✔️
- Good commit messages ✔️
- Professional README ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (chiztechnology) [reply schema.sql:74]: Hello  dear @alexander16108 🤝 
Thank you for your message.
Please find the screenshots below. 👇 

image

image

**Review by ShadyShawkat — APPROVED** [review]
## Hi @chiztechnology, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**